content/02.introduction.md

```diff
@@ -1,10 +1,14 @@
 ## Introduction
 
-- Importance of computation in biological research and scope.
-- Importance to reproducibility in computational biology.
-- Mentioning previous literature and some of their major takeaways and topics not fully covered in here (what kind of languages to learn, advanced text editors/IDEs,
-- The computational biology best-practices continuum: reproducibility at the personal level, sharing our research with others and over time.
-- Addressing the audience: computational biologists researchers – data analysts, workflow designers, software developers, mathematical modelers, etc.
-- Structure of this manuscript: mention Figure 1 and Figure 2.
-  - **Figure 1:** Funnel structure for better computational biology.
-  - **Figure 2:** Types of computational biology projects per level.
+Since Margaret Dayhoff pioneered the field of bioinformatics back in the decade of the sixties, the application of computational tools to the field of biology has vastly grown in scope and impact. Nowadays, biotechnological and biomedical research are routinely fed by the insights arising from novel computational approaches, machine learning algorithms and mathematical models. The ever increasing amount of biological data and the exponential growth in computing power will amplify this trend in the years to come.
+
+The use of computers to address biological matters encompasses a wide array of applications usually grouped under the terms of computational biology and bioinformatics. Although distinct definitions have been delineated for each one, here we will consider “computational biology” as the umbrella term for any application that involves the intersection of computing and modern biological data. As such, a computational biologist can be a bioinformatician, a statistician, a mathematical modeler, a data analyst, a data engineer, a software developer, and many others. In praxis, the modern computational biologist will be a “scientist of many hats”, taking on several of the duties listed above. But first and foremost, we will consider a computational biologist as a scientist whose ultimate goal is answering a biological question or addressing a need in the life sciences.
```

Comment:
Yes! The problem is that there are conflicting definitions about bioinformatics and computational biology.
- Here is the NIH working definition of these concept which deeply confuses me: http://www.binf.gmu.edu/jafri/math6390-bioinformatics/workingdef.pdf
- Here is a whole... article? that defines bioinformatics in a more specific way: https://pubmed.ncbi.nlm.nih.gov/11552348/
- Here is another confusing definition: https://www.medicaltechnologyschools.com/biotechnology/bioinformatics-vs-computational-biology
- And here is Nature's definition which basically stick them together: https://www.nature.com/subjects/computational-biology-and-bioinformatics#:~:text=Definition,predictions%20or%20discover%20new%20biology.
We can discuss this in our in-person meeting, because it's quite convoluted.

Reply:
Computational Biology has now been defined, and Bioinformatics will not be defined :)